packages/backend/src/tokens/clerkRequest.ts

```diff
@@ -0,0 +1,68 @@
+import { parse as parseCookies } from 'cookie';
+
+import { constants } from '../constants';
+
+export type WithClerkUrl<T> = T & {
+  /**
```

Comment:
❓ Do we still need this in the handshake world?

Reply:
We still need it for `authenticateRequest` and `redirectToSignIn`/`protect` from within `clerkMiddleware` and `authMiddleware`